sktime/classification/distance_based/_proximity_forest.py

```diff
@@ -131,9 +131,16 @@ def distance(instance_a, instance_b, **params):
 def distance_predefined_params(distance_measure, **params):
     """Conduct distance measurement with a predefined set of parameters.
 
-    :param distance_measure: the distance measure to use
-    :param params: the parameters to use in the distance measure
-    :returns: a distance measure with no parameters
+    Parameters
+    ----------
+    distance_measure: the distance measure to use, valid distance measures are
```

Comment:
is that always a string? Then I would write:

```
distance_measure: str, one of [insert the list]
    string code for the distance measure ot use
```

Reply:
It slipped under my radar this function receives a `callable` distance measure function.
Yeah, I will work on this in the future and ensure that the docstring is clearly explained.